telemetry/definitions/commonDefinitions.json

```diff
@@ -855,6 +855,51 @@
             "name": "userChoice",
             "type": "string",
             "description": "User selection from a predefined menu (not user-provided input)" 
+        },
+        {
+            "name": "codewhispererterminal_command",
+            "type": "string",
+            "description": "The CLI tool a completion was for"
+        },
+        {
+            "name": "codewhispererterminal_duration",
+            "type": "int",
+            "description": "A duration in milliseconds"
+        },
+        {
+            "name": "codewhispererterminal_duration",
```

Comment:
looks like `codewhispererterminal_duration` is redudant

Reply:
The duration type in our language represents milliseconds using (I think) 128 bit unsigned ints. The top level duration is a double. This shouldn't matter, but does matter in our generated code.

Reply:
nope, oversight